src/main/java/com/testjava/metrics_platform/TestJavaEventSender.java

```diff
@@ -18,38 +23,40 @@
 import com.google.gson.Gson;
 
 public class TestJavaEventSender implements EventSender {
+
+    private final Gson gson = new Gson();
```

Comment:
There is no need to recreate a new Gson serializer for each call to sendEvents, so moving it as an instance variable.